ros_connect.bash

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+echo "Be sure to call this script: source ./ros_connect {ROS_MASTER_IP} or source ./ros_connect {ROS_MASTER_IP} {ROS_TALKER_IP}"
```

Comment:
Also, please add a comment linking to the new [Computers with ROS](https://github.com/RoBorregos/Robocup-Home/wiki/Computers#running-ros-on-multiple-machines) wiki page